reflex/event.py

```diff
@@ -181,6 +181,18 @@ def debounce(self, delay_ms: int):
             event_actions={"debounce": delay_ms, **self.event_actions},
         )
 
+    @property
```

Comment:
`throttle` and `debounce` don't have the `@property` decorator but are defined similarly, is it just those two that don't need it?

Reply:
yeah this one is more along the lines of `prevent_default` and `stop_propagation`